### PR TITLE
[MP JWT] - Fix the Keycloak integration test in order to use Keycloak…

### PR DESCRIPTION
**Pull request by fabiobrz**
… 24.0, not latest

- Internal validation run: eap-8.x-microprofile-simple-face, 105

Please make sure your PR meets the following requirements:
- [x] Pull Request contains a description of the changes
- [x] Pull Request does not include fixes for multiple issues/topics
- [x] Code is formatted, imports ordered, code compiles and tests are passing
- [x] ~Link~ Reference to the passing job is provided
- [x] Code is self-descriptive and/or documented
- **N/A** Description of the tests scenarios is included (see #46)

### CODE REVIEW

File: microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java [hunk 1]
```diff
@@ -51,7 +51,7 @@ public class KeycloakIntegrationHighLevelScenarioTest {
     private static final String KEYCLOAK_ADMIN_PASSWORD = "password";
 
     public static Docker keycloakContainer = new Docker.Builder(KEYCLOAK_CONTAINER_NAME,
-            "quay.io/keycloak/keycloak:latest")
+            "quay.io/keycloak/keycloak:24.0")
```
Comment (jbliznak) [inline microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java:54]: with this we have come a full circle https://github.com/jboss-eap-qe/eap-microprofile-test-suite/pull/266

Comment (fabiobrz) [reply microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java:54]: Well, not actually full :)

Comment (fabiobrz) [reply microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java:54]: I agree that we could try and live with just latest though...

Comment (jbliznak) [reply microprofile-jwt/src/test/java/org/jboss/eap/qe/microprofile/jwt/keycloak/KeycloakIntegrationHighLevelScenarioTest.java:54]: yeah, but let's go with this quick fix while we plan to update later..

**Pull request merged**